pkg/provider/kubernetes/crd/kubernetes_http.go

```diff
@@ -380,11 +380,11 @@ func (c configBuilder) nameAndService(ctx context.Context, namespaceService stri
 			return "", nil, err
 		}
 
-		fullName := fullServiceName(svcCtx, namespace, service.Name, service.Port)
+		fullName := fullServiceName(svcCtx, namespace, service, service.Port)
```

Comment:
If we are passing in the service object, does it make sense to also pass in `service.Port` separately?

Reply:
No, because we use this function in another case:
https://github.com/containous/traefik/pull/6087/files#diff-8a53ad85b7fce7f8a74e0d7269162e3bR387